src/pages/Auth/LoginPage/components/LoginForm/index.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import useBooleanState from 'utils/hooks/state/useBooleanState';
+import { useLogin } from 'pages/Auth/LoginPage/hooks/useLogin';
+import BlindIcon from 'assets/svg/blind-icon.svg';
+import ShowIcon from 'assets/svg/show-icon.svg';
+import useLogger from 'utils/hooks/analytics/useLogger';
+import { cn } from '@bcsdlab/utils';
+import useMediaQuery from 'utils/hooks/layout/useMediaQuery';
+import styles from './LoginForm.module.scss';
+
+interface IClassUser {
+  userId: HTMLInputElement | null;
+  password: HTMLInputElement | null;
+}
+
+export default function LoginForm() {
+  const [isPasswordVisible, , , toggleIsPasswordVisible] = useBooleanState(false);
+  const [isAutoLoginFlag, , , toggleIsAutoLoginFlag] = useBooleanState(false);
+  const isMobile = useMediaQuery();
+  const submitLogin = useLogin({ isAutoLoginFlag });
+  const loginRef = React.useRef<IClassUser>({
+    userId: null,
+    password: null,
+  });
+  const onSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    const { userId, password } = loginRef.current;
+    submitLogin({
+      userId: userId ? userId.value : '',
+      password: password ? password.value : '',
+    });
+  };
+  const logger = useLogger();
```

Comment:
순서를 정돈하고 싶군요. ㅎㅎ
개인적으로 ref에 들어가는 키 이름도 조금 변경하고 싶습니다.

```suggestion
  const isMobile = useMediaQuery();
  const logger = useLogger();
  const submitLogin = useLogin({ isAutoLoginFlag });
  const [isPasswordVisible, , , toggleIsPasswordVisible] = useBooleanState(false);
  const [isAutoLoginFlag, , , toggleIsAutoLoginFlag] = useBooleanState(false);
  const loginRef = React.useRef<IClassUser>({ // 여기에 각각의 키에 적절한 subffix를 줘서 엘리먼트라는 느낌을 주고 싶네요.
    userId: null,
    password: null,
  });

  const onSubmit = (e: React.FormEvent<HTMLFormElement>) => {
    e.preventDefault();
    const { userId, password } = loginRef.current; // 그럼 여기 코드가 좀 더 정갈해질+가독성이 올라갈 것 같은 느낌?
    submitLogin({
      userId: userId ? userId.value : '',
      password: password ? password.value : '',
    });
  };
```

예시를 들자면 이렇게 써줄 것 같은 느낌이에요. 어떤가요?
```tsx
  const onSubmit = (e: React.FormEvent<HTMLFormElement>) => {
    e.preventDefault();
    const { userIdInput, passwordInput } = loginInputRef.current;
    const userId = userIdInput ? userIdInput.value : '';
    const password = passwordInput ? passwordInput.value : '';

    submitLogin({ userId, password });
  };
```

Reply:
lint 이슈로 `isAutoLoginFlag` 는 상단으로 올려서 사용해야 하기 때문에 `useBooleanState`로 정의한 상태를 가장 상단으로 올렸습니다.

말씀하신대로 `userIdInput` `passwordInput` 으로 사용하는 것이 훨씬 깔끔하고 보기에도 편한 것 같아요. 해당 부분 수정했습니다.
37373e31745306e366dae3510f1554dfa2afaf2b